pennylane_lightning/src/benchmarks/plot_gate_benchmark.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+import json
+import sys
+import numpy as np
+from pathlib import Path
+import matplotlib.pyplot as plt
+import argparse
+from collections import defaultdict
+
+from re import compile as re_compile
+
+plt.rc("font", family="sans-serif")
+plt.rc("text.latex", preamble=r"\usepackage{amsmath}")
+
+
+class DataProcessor:
```

Comment:
Would a more fitting name here be "BenchmarkProcessor" or "BenchmarkDataProcessor" ?

Reply:
Renamed!